pkg/drivers/vmware/vmware_windows.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2017 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vmware
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+func DhcpConfigFiles() string {
+	return `C:\ProgramData\VMware\vmnetdhcp.conf`
+}
+
+func DhcpLeaseFiles() string {
+	return `C:\ProgramData\VMware\vmnetdhcp.leases`
+}
+
+func SetUmask() {
+}
+
+func setVmwareCmd(cmd string) string {
+	cmd = cmd + ".exe"
+
+	if path, err := exec.LookPath(cmd); err == nil {
+		return path
+	}
+	for _, fp := range []string{
+		`C:\Program Files (x86)\VMware\VMware Workstation`,
```

Comment:
not guaranteed to be the case. might be possible to include the Uninstall stored data for this. eg. people might have the  application installed on the `D:` drive

Reply:
Hmm. Not a Windows expert. But you are right. People might install it some other places. Maybe get it in registry, or some other places. Will address it.

Reply:
done.